docs/specifications/tests/Consistency-TP/consistency05.md

```diff
@@ -1,49 +1,205 @@
-## CONSISTENCY05: Consistency between glue and authoritative data
+# CONSISTENCY05: Consistency between glue and authoritative data
 
-### Test case identifier
+## Test case identifier
 
-**CONSISTENCY05:** Consistency between glue and authoritative data
+**CONSISTENCY05**
 
-### Objective
+## Objective
 
 For name servers that have IP addresses listed as glue, the IP addresses must
-match the authoritative A and AAAA records for that host. This is an IANA
-requirement (https://www.iana.org/help/nameserver-requirements).
+match the authoritative A and AAAA records for that host. This is an IANA 
+[name server requirement].
+
+The objective of this test is to verify that the glue address records 
+in the delegation are consistent with authoritative data.
+
+## Inputs
+
+* "Child Zone" - The domain name to be tested.
+
+## Ordered description of steps to be taken to execute the test case
+1. Obtain the set of name server names (using [Method2]) and the IP 
```

Comment:
Obtain the set of glue records - i.e. glue name server names (using [Method2]) and the glue IP

Reply:
We could use the term "glue name server names" meaning the name server names of the NS in the delegation, but then we have to define that term elsewhere since it is not commonly defined (e.g. not found in RFC 7719. I think it is clearer to the delegation. I'll do an update.